crawler.go

```diff
@@ -58,6 +58,11 @@ type Crawler struct {
 	// Default is 10s.
 	DefaultRetryPeriod time.Duration
 
+	// Transport specifies the mechanism by which HTTP requests are made. Note that if populated, then this field would
+	// take precedence over other configs that determine the transport configuration e.g.: MaxConcurrentRequestsPerSite.
```

Comment:
Would it make sense to remove `MaxConcurrentRequestsPerSite` in our fork just to avoid confusion?

Reply:
I don't see us using it at any point so don't mind leaving it as it is.